desktop_version/CMakeLists.txt

```diff
@@ -169,6 +174,14 @@ endif()
 set(XML2_SRC
     ../third_party/tinyxml2/tinyxml2.cpp
 )
+set(FAUDIO_SRC
+	../third_party/FAudio/src/FAPOBase.c
```

Comment:
Woops, forgot this one... I _think_ this can be removed too, I would check locally first.

Reply:
Worked without as well. Removing